examples/basic_ops.rs

```diff
@@ -1,53 +1,64 @@
 #![allow(non_snake_case)]
-/// The purpose of this implementation is to provide support for one of the most 
-/// commonly used operations over EC which is Random Scalar Mul. 
-/// 
-/// This is one of the first steps to take on the implementation of algoritnms
-/// like Diffie-Hellman Key Exchange.
-/// 
-/// Let G be a point over Doppio, let k = Scalar random value over the Doppio Sub-group. 
+extern crate rand;
+/// The purpose of this implementation is to provide support for one of the most
+/// commonly used operations in Elliptic Curve Cryptography (ECC), which is Random Scalar Mul.
+///
+/// This is one of the first steps to take on the implementation of algorithms,
+/// such as  Diffie-Hellman Key Exchange.
+///
+/// Let G be a point over Sonny, let k = Scalar random value over the Sonny Sub-group.
 /// Let P = G*k;
-
 extern crate zerocaf;
-extern crate rand;
 
+use zerocaf::edwards::EdwardsPoint;
 use zerocaf::field::FieldElement;
 use zerocaf::scalar::Scalar;
-use zerocaf::edwards::EdwardsPoint;
 
-use rand::{Rng, thread_rng};
+use rand::{thread_rng, Rng};
 
 fn main() -> () {
-
-    // Let G be an `EdwardsPoint` which is a point over the Twisted Eds Extended Coordinates. 
+    // Let G be an `EdwardsPoint` which is a point over the Twisted Edwards Extended Coordinates.
     let G: EdwardsPoint = EdwardsPoint {
         X: FieldElement([23, 0, 0, 0, 0]),
-        Y: FieldElement([1664892896009688, 132583819244870, 812547420185263, 637811013879057, 13284180325998]),
```

Comment:
The same here applies to the comment on one of the previous one wrt the field element representation.

Reply:
I'll open a issue for refactoring the example. 

I'll add the ECDH implementation there as an example of how to use `RistrettoPoints` and also to update the `outdated` example that we have now there.